### PR TITLE
Week 02 day 04 mod 02

### PR DESCRIPTION
**Pull request by cameronjacoby**

### CODE REVIEW

**Comment by cameronjacoby** [comment]
@eerwitt @ben-manning No longer WIP; ready for review.

File: week-02-ajax-and-js-objects/day-04/module-02/readme.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+function Person(name) {
+  this.name = name;
+}
+
+Person.prototype.greet = function() {
+	return "Hello, my name is " + this.name;
+};
+
+function Student(name, course) {
+	this.name = name;
```
Comment (eerwitt) [inline week-02-ajax-and-js-objects/day-04/module-02/readme.md]: Not a big deal, but some indenting is 2sp and some is 4sp.

**Comment by eerwitt** [comment]
:+1: I think this will give great practice with all the OOP concepts.

File: week-02-ajax-and-js-objects/day-04/module-02/readme.md (hunk 1 above)
Comment (cameronjacoby) [inline week-02-ajax-and-js-objects/day-04/module-02/readme.md]: Thanks for catching that - all fixed!

**Pull request merged**